systemtests/setup_tests.py

```diff
@@ -27,44 +27,41 @@ def setup_tests(ctx):
         os.makedirs(ZIPS_DIR)
 
     click.echo("Generating systemtest data files ...")
-    try:
```

Comment:
Removing this try/except outer layer will cause exceptions to bubble all the way up and then Python will print the exception type, message, _and traceback_ which makes it easier to figure out what went wrong.